Guide/Constructive/Bonetrousle/README.md

```diff
@@ -62,6 +32,12 @@ formula above so that it does work. Note that the problem says that any combinat
 the needed number of noodles is a valid solution.
 </p>
 
+<p>
+It is important to keep in mind that for this problem some variable values could exceed
+the value of an int in Java. Also, System.out.println(), may be too slow for big inputs.
```

Comment:
How did you get the code to look like that? Online it says to use `code`, but that wasn't doing anything on my computer.